test/replication-luatest/gh_10836_crash_in_box_ctl_promote_test.lua

```diff
@@ -0,0 +1,121 @@
+local server = require("luatest.server")
+local replica_set = require("luatest.replica_set")
+local proxy = require("luatest.replica_proxy")
+local t = require("luatest")
+
+local g = t.group()
+
+g.before_all(function()
+    g.replicaset = replica_set:new({})
+
+    g.uri1 = server.build_listen_uri("server1", g.replicaset.id)
+    g.uri2 = server.build_listen_uri("server2", g.replicaset.id)
+    g.uri3 = server.build_listen_uri("server3", g.replicaset.id)
+
+    g.proxy_uri_1_to_2 = server.build_listen_uri(
+        "proxy_1_to_2", g.replicaset.id)
+    g.proxy_uri_1_to_3 = server.build_listen_uri(
+        "proxy_1_to_3", g.replicaset.id)
+    g.proxy_uri_2_to_1 = server.build_listen_uri(
+        "proxy_2_to_1", g.replicaset.id)
+    g.proxy_uri_3_to_1 = server.build_listen_uri(
+        "proxy_3_to_1", g.replicaset.id)
+
+    local box_cfg = {
+        election_mode = "manual",
+        replication_timeout = 0.1,
+        -- election_timeout = 0.1,
```

Comment:
Lets delete the commented out code. Feel free to resolve this comment on your own when done.